src/chatlog/content/notificationicon.h

```diff
@@ -31,7 +31,7 @@ class NotificationIcon : public QObject,  public ChatLineContent
 {
     Q_OBJECT
 public:
-    NotificationIcon(QSize size);
+    explicit NotificationIcon(QSize size);
```

Comment:
It's just like a const. Used for inability of assignment QSize to NotificationIcon. I think it is logical